cli/service_command.go

```diff
@@ -379,11 +382,40 @@ func (c *serviceCmd) infoAction(_ *fisk.ParseContext) error {
 		return err
 	}
 
+	var enfo *micro.EndpointInfo
+	var estats *micro.EndpointStats
+
+	if c.endpoint != "" {
+		for _, e := range nfo.Endpoints {
+			if c.endpoint == e.Name {
+				enfo = &e
+				break
+			}
+		}
+		if enfo == nil {
+			return fmt.Errorf("endpoint not found: %s", c.endpoint)
+		}
+
+		for _, s := range stats.Endpoints {
+			if s.Name == c.endpoint {
+				estats = s
+				break
+			}
```

Comment:
Thoughts on erroring out vs just not showing stats here? Currently I'm allowing for the stats to not be found for the endpoint and they just won't show.

Reply:
I do, but you're changing the output :)  Endpoints is an array so just remove the stuff not matching filter from it and rest of code doesn't need to change, unless I am missing something.

Prefer to keep the change surface minimal for now

Reply:
Ah yeah I see what you're saying. Will update :+1:

Reply:
Updated to just filter and not change output